app/lib/email_validator.rb

```diff
@@ -0,0 +1,11 @@
+class EmailValidator
+  include ActiveModel::Validations
+
+  attr_reader :email
+
+  def initialize(email)
+    @email = email
+  end
+
+  validates :email, presence: true, format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i }
```

Comment:
`URI::MailTo::EMAIL_REGEXP` nevieme pouzit?
https://stackoverflow.com/a/49925333/1167094

Reply:
@eronisko ten regex co navrhujes som tam mal, ale on matchne aj "tomas@radic" napriklad - to sa mi nepacilo, preto je tam tento.

Reply:
Dik za tu zmenu. Radsej by som tu bol benevolentny ako striktny -- nechceme byt v situacii ked sa niekto nebude vediet prihlasit len preto, ze maju nestandardny email.